cfgov/legacy/housing_counselor/geocoder.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import, unicode_literals
+
+from six import print_
+
+
+def geocode_counselors(counselors, **kwargs):
+    """Fill in missing latitude/longitude data for housing counselors."""
+    return ZipCodeBasedGeocoder(**kwargs).geocode(counselors)
+
+
+class ZipCodeBasedGeocoder(object):
+    """Fill in missing latitude/longitude data using zipcode locations.
```

Comment:
That's what I though too, but in practice this doesn't seem to happen right now. I was surprised and spent some time in this code, but it really does seem like [these lines](https://github.com/cfpb/django-hud/blob/master/hud_api_replace/management/commands/load_hud_data.py#L158) aren't ever executed right now (at least with the data I could experiment with). I suspect that because we do a radius-based search it doesn't return any counselors without coordinates.